### PR TITLE
Jb52535

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Just the last three commits belong to this PR

File: sdk-setup/src/sdk-assistant [hunk 1]
```diff
@@ -328,6 +334,10 @@ if [[ $OPT_ALL && ! $OPT_REGISTER ]]; then
     fail "The '--all' option can only be used with the 'register' command"
 fi
 
+if [[ $OPT_ALL && ! $OPT_REMOVE ]]; then
```
Comment (vigejolla) [inline sdk-setup/src/sdk-assistant:337]: Shouldn't the $OPT_ALL be $OPT_SNAPSHOTS_OF?

**Review by vigejolla — CHANGES_REQUESTED** [review]

File: sdk-setup/src/sdk-assistant (hunk 1 above)
Comment (martyone) [reply sdk-setup/src/sdk-assistant]: yup, fixed

**Review by vigejolla — APPROVED** [review]
LGTM

**Pull request merged**